libs/ngx-charts-on-fhir/src/lib/fhir-mappers/medication-request/simple-medication-mapper.service.ts

```diff
@@ -32,6 +32,7 @@ export class SimpleMedicationMapper implements Mapper<SimpleMedication> {
   constructor(@Inject(CATEGORY_SCALE_OPTIONS) private categoryScaleOptions: ScaleOptions<'category'>, private codeService: FhirCodeService) {}
   canMap = isMedication;
   map(resource: SimpleMedication): DataLayer<TimelineChartType, MedicationDataPoint[]> {
+    this.registerCustomPlugin();
```

Comment:
If this is called from the map function, it will re-register the plugin every time a medication resource is mapped. Can we call it from the constructor instead?